doc/DOCUMENTATION.org

```diff
@@ -442,6 +443,20 @@ manually from the template file =~/.emacs.d/core/templates/.spacemacs.template=
     $ cp ~/.emacs.d/core/templates/.spacemacs.template ~/.spacemacs
 #+end_src
 
+*** Alternative setup
+Since v0.104 you have the option of using =~/.spacemacs.d/init.el= for your
+dotfile instead of =~/.spacemacs=. If you want to use this option, simply move
+=~/.spacemacs= to =~/.spacemacs.d/init.el=. =~/.spacemacs= will alway takes
```

Comment:
a couple typos at the end of this line—it should probably be: `=~/.spacemacs= will always take`